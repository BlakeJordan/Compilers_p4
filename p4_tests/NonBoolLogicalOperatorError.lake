bool f() {
    int a;
    a = 5;
    return !a;
}