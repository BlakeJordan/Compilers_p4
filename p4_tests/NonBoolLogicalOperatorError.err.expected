4,12: Logical operator applied to non-bool operand
Type checking failed